docs/UserGuide.md

```diff
@@ -30,7 +30,8 @@ ContactsForGood (CFG) is a **desktop app for managing contacts, optimized for us
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add r/volunteer h/30 n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a 
+     contact of type `volunteer` and named `John Doe` to the Address Book.
```

Comment:
Hmm, I asked GPT about the English here and he says `named` is better

Reply:
Ah ok i see what you mean. I think I got thrown off because it kind of felt like you were specifying properties?

I think maybe "Adds a contact named `John Doe` with a `volunteer` role to the Address Book." would be clearer.

 Also, I think role would be more accurate than type here.

Reply:
Ok, have it rephrased